transmart-api-server/grails-app/services/org/transmart/api/server/KeycloakResourceService.groovy

```diff
@@ -0,0 +1,73 @@
+package org.transmart.api.server
+
+import grails.transaction.Transactional
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.*
+import org.springframework.util.LinkedMultiValueMap
+import org.springframework.util.MultiValueMap
+import org.springframework.web.client.RestTemplate
+
+@Transactional
```

Comment:
it does not seem to be needed.

Reply:
ah yes, it was added automatically, when I created a new server, I'll remove it